images/build/debian-iptables/distroless/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG GORUNNERIMAGE
+# Using directly debian base, as we are going to just copy the required binaries
+FROM debian:bullseye as build
+
+COPY stage-binary-and-deps.sh /
+COPY get-binaries-from-package.sh /
+COPY package-utils.sh /
+
+
+ARG STAGE_DIR="/opt/stage"
+RUN apt -y update && \
+    apt -y install bash curl && \
+    mkdir -p "${STAGE_DIR}" && \
+    /get-binaries-from-package.sh "${STAGE_DIR}" conntrack \
+    ebtables    \
+    ipset       \
+    iptables    \
+    kmod        && \
+    `# dash is the real Debian basic shell` \
+    /stage-binary-and-deps.sh /bin/dash "${STAGE_DIR}" && \
+    `# below binaries are used by iptables-wrapper-installer.sh` \
+    /stage-binary-and-deps.sh /bin/cat "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /bin/chmod "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /bin/grep "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /bin/ln "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /bin/rm "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /usr/bin/wc "${STAGE_DIR}"
```

Comment:
well, as we are "forking" the script, I can probably change this as well to loop over arguments and run only once